src/app/component/comments/components/comment-body/comment-body.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="comment-body-wrapper" *ngFor="let element of elements">
+<div class="comment-body-wrapper" *ngFor="let element of elements; index as i">
```

Comment:
Please fix alignment;
Check if you use "index as i";

Reply:
> Please fix alignment;
> Check if you use "index as i";

Fixed